syft/serde/serde.py

```diff
@@ -123,6 +123,12 @@
 
 ## SECTION: High Level Simplification Router
 def _force_full_simplify(obj: object) -> object:
+    """To force a full simplify genrally if the usual _simplify is not suitable.
+    
+    Args:
+    :param: the object
+    :return: the result from either _simplify or _forced_full_simplify on the object
```

Comment:
Docstring return should be presented as:

```
Returns:
     The result from either _simplify or _forced_full_simplify on the object.
```

Here's a nice docstring example: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html